tests/Unit/Domain/Test_CoordsToDifferentFrame.cpp

```diff
@@ -0,0 +1,82 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <array>
+#include <memory>
+#include <random>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/IndexType.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/CoordsToDifferentFrame.hpp"
+#include "Domain/Creators/RegisterDerivedWithCharm.hpp"
+#include "Domain/Creators/Sphere.hpp"
+#include "Domain/Creators/TimeDependence/RegisterDerivedWithCharm.hpp"
+#include "Domain/Creators/TimeDependence/Shape.hpp"
+#include "Domain/Domain.hpp"
+#include "Domain/FunctionsOfTime/RegisterDerivedWithCharm.hpp"
+#include "Domain/StrahlkorperTransformations.hpp"
+#include "NumericalAlgorithms/SphericalHarmonics/StrahlkorperFunctions.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace {
+template <typename SrcFrame, typename DestFrame>
+void test_coords_to_different_frame() {
+  const size_t grid_points_each_dimension = 5;
+
+  const tnsr::I<DataVector, 3, SrcFrame> src_center{DataVector{0.03}};
+
+  const std::vector<double> radial_partitioning{};
+  std::vector<domain::CoordinateMaps::Distribution> radial_distribution{
+      domain::CoordinateMaps::Distribution::Linear};
+  std::unique_ptr<domain::creators::Sphere> domain_creator;
+
+  domain_creator =
```

Comment:
combine into one line and you don't need the `unique_ptr`: `const domain::creators::Sphere domain_creator{...}`